experimental/fairseq_moe_docker_slurm/environment/context/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG FROM_IMAGE_NAME=nvcr.io/nvidia/pytorch:21.10-py3
+  
+FROM ${FROM_IMAGE_NAME}
+
+ENV NCCL_DEBUG="WARN"
+ENV UCX_TLS="tcp"
+ENV UCX_NET_DEVICES="eth0"
+ENV NCCL_SOCKET_IFNAME="eth0"
```

Comment:
Is there an option to use IB here?

Reply:
I thought about the same thing. These variable settings are inherited from the original setup. My guess is that these communications do not involve RDMA so diverted to ethernet port. @garvct can comment on it.

Reply:
Yeah, I've seen it elsewhere and I was immediately curious about it.  Even without RDMA, IB will have high bandwidth and lower latency than using eth0.